.chloggen/feat_rename-awslogsencodingextension-format-names.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

```suggestion
change_type: deprecation
```

It's more important to highlight the deprecation than the enhancement.